### PR TITLE
Add carnivorous potted plant for gingerbread city and the menagerie

### PR DESCRIPTION
**Pull request by s-k-z**
Free kills from the carnivorous potted plant are even better than free runs here so we get extra value out of equipping it in these locations.

### CODE REVIEW

File: src/fights.ts [hunk 1]
```diff
@@ -1335,6 +1335,14 @@ const freeRunFightSources = [
       ) {
         set("_gingerbreadCityTurns", 1 + get("_gingerbreadCityTurns"));
       }
+    },
+    {
+      requirements: () => [
+        new Requirement([], {
+          // eslint-disable-next-line libram/verify-constants
+          forceEquip: $items`carnivorous potted plant`.filter((item) => have(item)),
```
Comment (horrible-little-slime) [inline src/fights.ts:1343]: It looks like the plugin just upgraded; you may not need this

**Review by horrible-little-slime — CHANGES_REQUESTED** [review]
we need to update our macros to take advantage of this or else it's not actually reaping any benefits.

File: src/fights.ts (hunk 1 above)
Comment (s-k-z) [reply src/fights.ts:1343]: Sadly not, upgraded around 11 hours ago and KoLMafia only added the data around 2 hours ago.

**Comment by s-k-z** [comment]
> we need to update our macros to take advantage of this or else it's not actually reaping any benefits.

I don't think so. It should end combat immediately and just advance delay and grant items/meat.

**Pull request merged**